src/math/ColorSpan.js

```diff
@@ -55,11 +55,13 @@ export default class ColorSpan extends Span {
  * @return {?ColorSpan}
  */
 export const createColorSpan = colors => {
-  if (!colors) {
-    return null;
+  if (colors === undefined) {
```

Comment:
Maybe we need a utility to check if the arg is a valid hexadecimal? Something like 

```
if (!isHex(colors)) {
    console.warn(`Invalid color passed to createColorSpan:${colors}. Defaulting to 'random'.`)
    colors = 'random';
}
```

Because I guess the original issue was that passing `0` or `0x0` is failing. I don't know if `0` is a valid hex though